packages/core/src/__tests__/kernel-account.test.ts

```diff
@@ -0,0 +1,199 @@
+import {KernelHdConfig, KernelHdWalletProvider} from "../provider/implementation/kernel-hd-wallet";
+import {concatHex, encodeAbiParameters, Hex, parseEther} from "viem";
+import {polygonMumbai} from "viem/chains";
+import {mnemonicToAccount} from "viem/accounts";
+import {parseAbiParameters} from "abitype";
+
+
+describe("Kernel Account Tests", () => {
+    const chain = polygonMumbai
+    const config: KernelHdConfig = {
+        mnemonic: process.env.OWNER_MNEMONIC,
```

Comment:
currently working through a way to test this stuff without requiring an environment specific mnemonic (these tests definitely won't pass for me since I'm using a different mnemonic from y'all)

Reply:
Will push some more changes today which would allow tests tests to be run irrespective of wallet

Reply:
This is sorted now. You should be able to run test cases irrespective of wallet @moldy530